packages/build-utils/src/workspaces/workspace-managers.ts

```diff
@@ -0,0 +1,63 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of workspace managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { workspaceManagers as frameworkList } from '@vercel/build-utils/workspaces'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
+ */
+export const workspaceManagers = [
+  {
+    name: 'yarn',
```

Comment:
```suggestion
    name: 'Yarn',
```

Yarn is one of the few that has a capital letter